<ui version="4.0" >
 <class>WdgSumieOptions</class>
 <widget class="QWidget" name="WdgSumieOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Ink Depletion Curve</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="KCurve" native="1" name="inkCurve" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="brushGroup" >
     <property name="title" >
      <string>Brush Settings</string>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>120</x>
        <y>20</y>
        <width>226</width>
        <height>99</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="1" column="1" >
          <widget class="QSpinBox" name="radiusSpinBox" >
           <property name="value" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3" >
          <widget class="QDoubleSpinBox" name="sigmaSpinBox" >
           <property name="value" >
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QLabel" name="sigmaLabel" >
           <property name="text" >
            <string>sigma</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2" >
          <widget class="QRadioButton" name="oneDimBrushBtn" >
           <property name="text" >
            <string>1D</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="radiusLabel" >
           <property name="text" >
            <string>radius</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2" >
          <widget class="QRadioButton" name="twoDimBrushBtn" >
           <property name="text" >
            <string>2D</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>71</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <widget class="QLabel" name="inkLabel" >
           <property name="text" >
            <string>Ink Amount</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QSpinBox" name="inkAmountSpinBox" >
           <property name="maximum" >
            <number>10000</number>
           </property>
           <property name="value" >
            <number>4096</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="mousePressureCBox" >
      <property name="geometry" >
       <rect>
        <x>120</x>
        <y>120</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Mouse pressure</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KCurve</class>
   <extends>QWidget</extends>
   <header>widgets/kcurve.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
